### PR TITLE
Adds a `Windows Desktop Service` section to scaling

### LINKED ISSUE #14957: Desktop Access: document sizing guidelines for Windows Desktop Service
Document rough sizing guidelines for Windows Desktop Service.

Test with a specific Window size (1080p?) and measure memory usage for a single session as well as a couple concurrent sessions.

### PR DESCRIPTION
**Pull request by ibeckermayer**
Also adds a link in the windows desktop reference page, as well as including the CLI in the reference page bullet list.

Based on the reasoning [documented here](https://docs.google.com/document/d/1HsPgN2K6HU1X9iFZ_c9g1glrj4lMh1pXZsS_U1yo2iQ/edit?usp=sharing).

Closes #14957

### CODE REVIEW

File: docs/pages/management/operations/scaling.mdx [hunk 1]
```diff
@@ -124,4 +124,37 @@ has default limits of 4CPU and 4Gi memory.
...
+
+Windows Desktop sessions can vary greatly in resource usage depending on the applications being used. The primary
+factor affecting resource usage per session is how often the screen is updated. For example, a session playing a video
+in full screen mode will consume significantly more resources than a session where the user is typing in a text editor.
+
+We measured the resource usage of sessions playing fullscreen videos to get the worst-case estimate for resource requirements.
+We then inferred resource requirements for more standard use cases on the basis of those measurements.
+
+**Worst Case:**
+- 1/12 vCPU per concurrent session
```
Comment (probakowski) [inline docs/pages/management/operations/scaling.mdx:139]: Consider using rounder number here, e.g. `1/10`, `50MB`, `1/200`

**Review by probakowski — APPROVED** [review]
Great work 👍

**Review by ptgott — APPROVED** [review]

**Review by rosstimothy — APPROVED** [review]

File: docs/pages/management/operations/scaling.mdx (hunk 1 above)
Comment (ibeckermayer) [reply docs/pages/management/operations/scaling.mdx:139]: `1/12` is more accurate. It also feels "round-ish" to my imperial system sensibilities :)

**Review by zmb3 — APPROVED** [review]

**Comment by zmb3** [comment]
@ibeckermayer can you backport this to branch/v15?

**Pull request merged**